src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -342,17 +361,14 @@ function drush_jcms_notifications_message_import($lrp = FALSE) {
           case 'metrics':
             // Process article views-downloads metrics sqs items.
             $body = $message->getMessage();
-            if ($body['contentType'] == 'article' && $body['metric'] == 'views-downloads') {
-              $articleVersions = $article_service->getArticleVersions($id);
-              if ($articleVersions->getAction() == ArticleVersions::WRITE) {
-                // Make sure article has been created.
-                $node = $article_crud_service->crudArticle($articleVersions);
-                $metrics = $metrics_service->getArticleMetrics($id);
-                if ((int) $node->get('field_page_views')->getString() != $metrics->getPageViews()) {
-                  $logger->info('Adjusted metrics', ['id' => $id, 'metrics' => $metrics->getPageViews()]);
-                  $node->set('field_page_views', $metrics->getPageViews());
-                  $node->save();
-                }
+            if ($body['contentType'] === 'article' && $body['metric'] === 'views-downloads' && $nid = $article_crud_service->getNodeIdByArticleId($id)) {
```

Comment:
the `$nid = $article_crud_service->getNodeIdByArticleId($id)` condition checks that the article has been created in Journal CMS. If it hasn't we can discard this message and wait for knowledge of the article.